utam-core/src/main/java/utam/core/driver/Driver.java

```diff
@@ -151,4 +155,16 @@ public interface Driver {
    * Navigates forward in the current browsing context (mobile or web)
    */
   void forward();
+
+  String getWindowHandle();
+
+  Set<String> getWindowHandles();
+
+  WebDriver.TargetLocator switchTo();
+
+  Rect getRect();
+
+  void setRect(Rect rect);
+
+  void close();
```

Comment:
each method should have Javadoc. And do we really need all this as public methods? Seems weird.